### PR TITLE
[SalesForce] Added configable instance url

### PR DESCRIPTION
**Pull request by ronnievisser**
possibility to use for sandboxes and production environments

### CODE REVIEW

**Comment by faustbrian** [comment]
Please move the private method to the end of the class.

File: src/SalesForce/Provider.php [hunk 1]
```diff
@@ -23,12 +23,28 @@ class Provider extends AbstractProvider implements ProviderInterface
...
+     *
+     * @return string Salesforce base URL
+     */
+    private function getInstanceURL()
+    {
+        $fromConfig = config('services.salesforce.instance_url');
+        if (! is_null($fromConfig)) {
+            return $fromConfig;
+        }
+
```
Comment (faustbrian) [inline src/SalesForce/Provider.php]: Move this to the end of the file since the method is `private`.

File: src/SalesForce/Provider.php [hunk 2]
```diff
@@ -23,12 +23,28 @@ class Provider extends AbstractProvider implements ProviderInterface
     protected $scopeSeparator = ' ';
 
     /**
+     * Get the instance URL from config
+     * If not available default to production.
+     *
+     * @return string Salesforce base URL
+     */
+    private function getInstanceURL()
```
Comment (faustbrian) [inline src/SalesForce/Provider.php]: Move this to the end of the file since the method is private.

**Comment by ronnievisser** [comment]
propably a stupid question but "socialiteproviders/salesforce": "^2.0", in my composer.json file. will this update be reflected there? or do i need to load some other package to get his working?

**Comment by faustbrian** [comment]
@ronnievisser https://github.com/SocialiteProviders/SalesForce/releases/tag/v2.0.1

**Comment by ronnievisser** [comment]
I did that but didn't get any update. I removed and installed again and now all is ok!

**Comment by faustbrian** [comment]
It takes a few minutes for packagist.org <http://packagist.org/> to distribute the update, no need to reinstall a package if it doesn’t update instantly.

**Pull request merged**